packages/plugins/components/src/hooks/FeatureFlags/FeatureFlags.tsx

```diff
@@ -1,13 +1,16 @@
 /**
  * Feature Flag provider - allows a multi-stage development.
  */
+// eslint-disable-next-line import/no-duplicates
 import * as React from 'react';
 import { createContext, useCallback, useContext, useEffect, useState } from 'react';
-import { isDevEnv, isTestEnv } from 'common/env';
+// eslint-disable-next-line import/no-duplicates
+import { isDevEnv, isTestEnv } from '../../Utils';
```

Comment:
I don't see duplicates of either line - tried removing disabler locally, and no lint complains for me